src/SignalR/common/Shared/ISystemClock.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Internal
+{
+    internal interface ISystemClock
```

Comment:
Can we share this stuff with Kestrel?

Reply:
Probably, I did just copy it from Kestrel.